app/ante/eth.go

```diff
@@ -41,43 +42,35 @@ import (
 // - from address is empty
 // - account balance is lower than the transaction cost
 func VerifyEthAccount(
-	ctx sdk.Context, tx sdk.Tx,
+	ctx sdk.Context,
+	msgEthTx *evmtypes.MsgEthereumTx,
```

Comment:
What if we just change `sdk.Tx` to `[]*evmtypes.MsgEthereumTx`, that might need less code changes.